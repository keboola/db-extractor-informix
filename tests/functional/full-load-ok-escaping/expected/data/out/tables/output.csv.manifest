{
    "destination": "output",
    "incremental": false,
    "metadata": [
        {
            "key": "KBC.name",
            "value": "escaping"
        },
        {
            "key": "KBC.sanitizedName",
            "value": "escaping"
        },
        {
            "key": "KBC.schema",
            "value": "%A"
        },
        {
            "key": "KBC.type",
            "value": "table"
        }
    ],
    "column_metadata": {
        "id": %A,
        "value": %A
    },
    "columns": [
        "id",
        "value"
    ]
}
