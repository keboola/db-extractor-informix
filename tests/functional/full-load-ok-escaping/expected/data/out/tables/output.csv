"1",""""""""
"2","'''"
"3","```"
"4","1234úěščřžýáíé+!@#$%^&*()"
"5","	a
b
c	d"
